src/services/sendClientFcmToken.ts

```diff
@@ -9,7 +9,7 @@ type requestProps = {
 };
 
 const sendClientFcmToken = async ({ fcmToken, data }: requestProps) => {
-  await request('https://api2.wagent.app/services/send-notification', {
+  await request('https://api.wagent.app/services/send-notification', {
     method: 'POST',
     body: JSON.stringify({
       token: fcmToken,
```

Comment:
Use the new router (described in Swagger) and only call this when user is signed in

Reply:
This will resolve in next PR